### ISSUE TITLE
Label-visible content mismatch 'Admin' and 'Edit your account' in sidebar

### ISSUE THREAD

**Issue by thibaudcolas** [wagtail/wagtail#11372]
Initially reported by @albinazs as part of [WCAG 2.2 AAA* audit of the Wagtail admin – Nov 2023](https://github.com/wagtail/wagtail/discussions/11180).

In the sidebar, _Label-visible content mismatch 'Admin' and 'Edit your account'_

This is failure of _2.5.3 Label in Name_:

> When an interactive control has a name on screen, make sure that assistive technologies (like Voice Control) know it by the same name (or a name that includes the name on screen)

Code in question:

```html
<button class="[…]" title="Edit your account" aria-label="Edit your account" aria-haspopup="menu" aria-expanded="false" type="button">
```

---

## Solution

We should avoid `aria-label` here, and instead rely on the visible text. On `title` – I’m not sure how it contributes exactly, so might need removing, or be good to keep. We’d need to refer to [accessible name and description computation rules](https://www.w3.org/TR/accname-1.2/) and assistive tech support as part of devising the fix.

**Comment by nandini584** [comment]
I would like to work on this issue, could you provide any other info regarding it? Thankyou.

**Comment by noobCoderVP** [comment]
I would like to fix this issue following accessibility rules. Can you assign me this issue?

**Comment by zerolab** [comment]
@noobCoderVP thank you for your interest. We don't assign issues outside the core team. Please give https://docs.wagtail.org/en/stable/contributing/first_contribution_guide.html a read. The short version - always check the issue carefully, especially for any signs of an existing PR. If there's nothing, and noone else mentioned they are working on it, just go for it (ideally with a note that you are doing so)

**Comment by nandini584** [comment]
Thankyou so much. I am working on this issue then.

**Comment by noobCoderVP** [comment]
Thank you @zerolab! I will follow the contribution guidelines and try to resolve the issue.
